exonum/src/node/mod.rs

```diff
@@ -449,6 +498,7 @@ impl NodeHandler {
             channel: sender,
             peer_discovery: config.peer_discovery,
             is_enabled,
+            node_role,
```

Comment:
What happens with `node_role` if this node will become validator at runtime?

Reply:
Now: it just will forget to update `node_role` in `NodeHandler`, but will update `node_role` in `ApiState`.